apps/extension/src/provider/Signer.ts

```diff
@@ -20,17 +20,16 @@ import {
   TxProps,
   TxMsgValue,
   Schema,
-  SupportedTx,
   SubmitVoteProposalProps,
   SubmitVoteProposalMsgValue,
 } from "@namada/types";
-import { TxType } from "@namada/shared";
+import { TxType, SupportedTx } from "@namada/shared";
 
 export class Signer implements ISigner {
   constructor(
     protected readonly chainId: string,
     private readonly _namada: Namada
-  ) { }
```

Comment:
Sorry that my IDE format-on-save is adding this space. it does it about half of the time. It's finicky and super annoying. I need to fix that!

Reply:
No worries :)